src/emulator/host/src/host.cpp

```diff
@@ -235,9 +235,11 @@ bool init(HostState &state, Config cfg) {
                 if (vfs::read_app_file(params, state.pref_path, state.io.title_id, "sce_sys/param.sfo")) {
                     SfoFile sfo_handle;
                     load_sfo(sfo_handle, params);
+                    find_data(state.game_version, sfo_handle, "APP_VER");
```

Comment:
nit: put this line below TITLE, for consistency

Reply:
oki, i have just set it before because on game list and on sfo is writed before title